api/src/modules/impact/dto/impact-table.dto.ts

```diff
@@ -161,6 +163,17 @@ export class GetRankedImpactTableDto extends BaseImpactTableDto {
   @IsUUID(4)
   scenarioId?: string;
 
+  @ApiPropertyOptional({
+    description:
+      'Optional property for groupBy region and chart view to choose level of admin regions to show',
```

Comment:
I think the msg is not correct as the grouping depends on the selected entity, so its not tied to 'admin-regions'

I would say something like:
"Select the depth of the tree for grouping elements, defaults to...."